filebeat/module/apache2/access/_meta/fields.yml

```diff
@@ -85,10 +85,11 @@
           type: keyword
           description: >
             The name of the operating system.
-        - name: raw
+        - name: original
           type: text
```

Comment:
As we set `index: false` it should not matter here what type is defined. In ECS it seems we put `keyword`. Only reason I mention this is we should check later what shows up in the docs for non indexed fields.

Reply:
Got it. Thanks for the info.